apps/intellij/src/main/kotlin/dev/nx/console/generate/NxGenerateService.kt

```diff
@@ -118,3 +152,13 @@ class NxGenerateService(val project: Project) {
             project.getService(NxGenerateService::class.java)
     }
 }
+
+private fun matchWithWildcards(text: String, pattern: String): Boolean {
+    val regexFromPattern =
```

Comment:
I wonder if we can use some kind of matching library rather than do this regex ourselves.

Reply:
I looked around for a bit and didn't find anything that looked good - not sure if it's worth the effort.

Reply:
yea, good point